SubBox/app/src/main/java/com/joelimyx/subbox/Detail/DetailFragment.java

```diff
@@ -0,0 +1,131 @@
+package com.joelimyx.subbox.detail;
+
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.design.widget.Snackbar;
+import android.support.v4.app.Fragment;
+import android.text.TextUtils;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.joelimyx.subbox.R;
+import com.joelimyx.subbox.Classes.SubBox;
+import com.joelimyx.subbox.dbassethelper.SubBoxHelper;
+import com.squareup.picasso.Picasso;
+
+import java.text.NumberFormat;
+import java.util.Locale;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+
+public class DetailFragment extends Fragment {
+    @BindView(R.id.detail_image) ImageView detailImage;
+    @BindView(R.id.title_text) TextView mTitleText;
+    @BindView(R.id.detail_price_text) TextView mPriceText;
+    @BindView(R.id.detail_text) TextView mDetailText;
+    @BindView(R.id.detail_button) Button mDetailButton;
+    private static final String ARG_PARAM1 = "param1";
```

Comment:
Give this constant a more descriptive name. Also, give the parameter for `newInstance()` below a more descriptive name.